metrics/web/lib/auth/presentation/widgets/login_form.dart

```diff
@@ -0,0 +1,120 @@
+import 'package:flutter/material.dart';
+import 'package:metrics/auth/presentation/state/auth_notifier.dart';
+import 'package:metrics/auth/presentation/strings/auth_strings.dart';
+import 'package:metrics/auth/presentation/validators/email_validator.dart';
+import 'package:metrics/auth/presentation/validators/password_validator.dart';
+import 'package:metrics/base/presentation/widgets/svg_image.dart';
+import 'package:metrics/base/presentation/widgets/tappable_area.dart';
+import 'package:metrics/common/presentation/button/widgets/metrics_positive_button.dart';
+import 'package:metrics/common/presentation/metrics_theme/widgets/metrics_theme.dart';
+import 'package:metrics/common/presentation/widgets/metrics_text_form_field.dart';
+import 'package:provider/provider.dart';
+
+/// A widget that displays the email and password sign in form.
+class LoginForm extends StatefulWidget {
```

Comment:
Let's think about how to rename it as using `LoginForm` within `AuthForm` is a bit confusing. Maybe we should rename `AuthForm` or get rid of it at all and inline it into the `LoginPage`? 

Maybe we can leave the `AuthForm` to don't make the `LoginPage` more complex. Instead, the `LoginForm` could be `PasswordSignInOption` so the `AuthForm` contains sign in options available: 
```dart
        return Column(
          mainAxisAlignment: MainAxisAlignment.center,
          crossAxisAlignment: CrossAxisAlignment.stretch,
          children: <Widget>[
            if (isLoginFormEnabled) const PasswordSignInOption(),
            SignInOptionButton(
              strategy: GoogleSignInOptionAppearanceStrategy(),
            ),
          ],
        );
```

cc @illia-romanenko